signupForm.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
```

Comment:
The above lines are getting repeated in all the html files.  If i need to change the bootsrap version then i willl have to go and modify every html file. This is not good.    check this 

https://www.digitalocean.com/community/tutorials/how-to-use-ejs-to-tem plate-your-node-application

Reply:
Sure I will check that link